### DISCUSSION TITLE
Error when making a page

### DISCUSSION THREAD

**Discussion by ARR4NN** [requarks/wiki#6379] [Error / Bug Report] [answered]
warn: Error when running job render-page: EACCES: permission denied, open '/var/www/RonanHost/wiki/data/cache/b29b5d2ce62e55412776ab98f05631e0aa96597b.bin'

**Comment by NGPixel** [comment, marked as answer]
Check your file system permissions and make sure the node process has write access to the `data` folder (and everything under it)
